src/components/MaterialUI/DataDisplay/PredefinedElements/InformationItem.js

```diff
@@ -53,15 +53,15 @@ const useStyles = makeStyles(theme => ({
 	},
 }));
 
-const InformationItemChildren = ({ classes, children, showNotAvailable }) => {
+const InformationItemChildren = ({ classes, children, showNotAvailable, isMaxLineCountEnabled = true }) => {
 	const { formatMessage } = useIntl();
 
 	if (isReactComponent(children)) {
 		return children;
 	}
 
 	const multipleLinesTextProps = new TextProps();
-	multipleLinesTextProps.set(TextProps.propNames.lineCount, 2);
+	if (isMaxLineCountEnabled) multipleLinesTextProps.set(TextProps.propNames.lineCount, 2);
```

Comment:
I can see a future where we need to set the number of lines... this works for your PR, but maybe we should have a default max lines?

Reply:
Interesting. Although I can see the ground work for that is already present, I would consider this as being a separate feature and that would have to be done in a separate PR.